src/components/Dashboard/ActionToolbar/ActionToolbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {ReactElement} from 'react'
+import styles from './ActionToolbar.module.css'
+import saveIcon from './save-icon.png'
+import undoIcon from './undo-icon.png'
+import redoIcon from './redo-icon.png'
+import clearIcon from './clear-icon.png'
+
+type Props = {
+  editAction: () => void, clearAction: () => void, undoAction: () => void, redoAction: () => void
+}
+
+export const ActionToolbar = ({editAction, clearAction, undoAction, redoAction}: Props): ReactElement => (
+    <div role="toolbar" className={styles.tools}>
```

Comment:
I think we should be careful of adding aria roles unless we fulfil all the functionality that is expected for that role - i.e. for `toolbar` we need to implement focus management and keyboard interactions within the toolbar - see [link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role). I would prefer it if we could either find a html element with the correct semantics already or failing that find a different way to test, worst case scenario we can always use a [test-id](https://testing-library.com/docs/queries/bytestid/) (although this should be the last resort)